docs/chapters/developer_guide.tex

```diff
@@ -0,0 +1,176 @@
+\chapter{Developer Guide}
+
+\section{Introduction}
+
+This chapter gives an overview of how \dfastmi can be setup for development.
+
+The development setup and configuration is done for usage of the visual studio code IDE. More information about visual studio code can be found on the internet.
```

Comment:
maybe add  the link instead of saying "on the internet"?
https://code.visualstudio.com/

Reply:
agree, processed